### ISSUE TITLE
2.7.2100 version gets permissions denied when performing publish

### ISSUE THREAD

**Issue by ccwilkins** [Azure/azure-functions-core-tools#1779]
This new version appears to be bad and also does not appear in apt-get. 2.7.1948 from apt-get works but as soon as I went to 2.7.2100 I get the error below when publishing.

`func azure functionapp publish {functionAppName}`

Error returned:

```
[2020-01-23T12:41:27.255Z] System.ComponentModel.Win32Exception (13): Permission denied
[2020-01-23T12:41:27.255Z]    at Azure.Functions.Cli.Helpers.RetryHelper.Retry(Func`1 func, Int32 retryCount, TimeSpan retryDelay, Boolean displayError) in D:\a\1\s\src\Azure.Functions.Cli\Helpers\RetryHelper.cs:line 29
[2020-01-23T12:41:27.255Z]    at Azure.Functions.Cli.Actions.AzureActions.PublishFunctionAppAction.PublishZipDeploy(Site functionApp, Func`1 zipFileFactory) in D:\a\1\s\src\Azure.Functions.Cli\Actions\AzureActions\PublishFunctionAppAction.cs:line 682
[2020-01-23T12:41:27.255Z]    at Azure.Functions.Cli.Actions.AzureActions.PublishFunctionAppAction.PublishRunFromPackageLocal(Site functionApp, Func`1 zipFileFactory) in D:\a\1\s\src\Azure.Functions.Cli\Actions\AzureActions\PublishFunctionAppAction.cs:line 500
[2020-01-23T12:41:27.255Z]    at Azure.Functions.Cli.Actions.AzureActions.PublishFunctionAppAction.PublishFunctionApp(Site functionApp, GitIgnoreParser ignoreParser, IDictionary`2 additionalAppSettings) in D:\a\1\s\src\Azure.Functions.Cli\Actions\AzureActions\PublishFunctionAppAction.cs:line 320
[2020-01-23T12:41:27.255Z]    at Azure.Functions.Cli.Actions.AzureActions.PublishFunctionAppAction.RunAsync() in D:\a\1\s\src\Azure.Functions.Cli\Actions\AzureActions\PublishFunctionAppAction.cs:line 173
[2020-01-23T12:41:27.255Z]    at Azure.Functions.Cli.ConsoleApp.RunAsync[T](String[] args, IContainer container) in D:\a\1\s\src\Azure.Functions.Cli\ConsoleApp.cs:line 66
```

**Comment by dariagrigoriu** [comment]
@mhoeger do you happen to know if this was added to apt?

**Comment by mhoeger** [comment]
Thanks Daria!

@ccwilkins - we haven't released this to pre-release yet and it's still just undergoing internal testing. Thanks for the early heads up!

**Comment by ccwilkins** [comment]
Hi Marie,

Thank you for the update. Curious, how do I tell if a version is ready for usage here on the Github page? When I saw the release I assumed that it was ready for wide spread use. There is no indication that it is a version under test.

**Comment by mhoeger** [comment]
We just had the same discussion over here (cc: @yojagad )
In the future, we'll change the release order to not make the GitHub release until it's being released officially, but for this release, it's ready for wide spread use when it's available on npm, homebrew, and apt.

**Comment by ccwilkins** [comment]
Thanks for the update. So what is the latest recommended release of 2.7.X?

**Comment by ankitkumarr** [comment]
The latest version released is 2.7.2184

The permission denied was an issue with our last release. This should now be fixed. Closing as resolved in 2.7.2184

**Comment by gsshyam** [comment]
cloud shell still have old version
